src/PostList.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+type Props = {
+  posts: Post[];
+};
+
```

Comment:
add check for postList length. if array is empty return some message "Ooops, you dont have any post yet". and CTA button wich redirect customer to 'Add new post' page.